knpy/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
To not delay this review further, the general build process has not been reviewed. Although based on my experience of trying to do it on a Windows machine (and that you had to modify this file), it is not ready for review. If really needed, PR could be merged without proper building documentation and process.

Reply:
I think let us proceed with the merge and revisit this problem at a later date.